From 5392cd3c013bb43155b724a38ef79deeefa20169 Mon Sep 17 00:00:00 2001
From: Urja Rannikko <urjaman@gmail.com>
Date: Fri, 23 Apr 2021 14:15:13 +0300
Subject: [PATCH] package/links: graphics mode does not depend on DirectFB or
 X11

The "fb" framebuffer driver does not have any special dependencies.

Signed-off-by: Urja Rannikko <urjaman@gmail.com>
---
 package/links/Config.in | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/package/links/Config.in b/package/links/Config.in
index 713f53da4e..64683ac772 100644
--- a/package/links/Config.in
+++ b/package/links/Config.in
@@ -10,11 +10,10 @@ if BR2_PACKAGE_LINKS
 
 config BR2_PACKAGE_LINKS_GRAPHICS
 	bool "use graphics"
-	depends on BR2_PACKAGE_DIRECTFB || BR2_PACKAGE_XORG7
 	select BR2_PACKAGE_XLIB_LIBXT if BR2_PACKAGE_XORG7
 	select BR2_PACKAGE_LIBPNG
 	help
-	  Enable graphics output for DirectFB or X11.
+	  Enable graphics output for framebuffer, DirectFB or X11.
 	  Otherwise links will be text-mode only.
 
 endif
-- 
2.31.1

